index.html

```diff
@@ -0,0 +1,172 @@
+        Main program
+      </h3>
+      <hr>
+      <div class="container progs" id="prog-list">
+        <div class="pro-item">
+          <div class="prog-img-ph">
+            <img src="images/lecture.png" alt="" class="prog-img">
+          </div>
+          <div class="topic">
+            <h4 class="clr-2 head4">Lecture</h4>
+          </div>
+          <p class="desc">
+            Listen from the experts of various parts of India, About Stock Market and digital world.
+          </p>
+        </div>
+        <div class="pro-item">
+          <div class="prog-img-ph">
+            <img src="images/exhibition.png" alt="" class="prog-img">
+          </div>
+          <div class="topic">
+            <h4 class="clr-2 head4">DW Showcase</h4>
+          </div>
+          <p class="desc">
+            Presentations,financial education and awardness by Digital world.
+          </p>
+        </div>
+        <div class="pro-item">
+          <div class="prog-img-ph">
+            <img src="images/forum.png" alt="" class="prog-img">
+          </div>
+          <div class="topic">
+            <h4 class="clr-2 head4">Forum</h4>
+          </div>
+          <p class="desc">
+            Sharing Stock Market expert guidance
+            and Technical analysis.
+          </p>
+        </div>
+        <div class="pro-item">
+          <div class="prog-img-ph">
+            <img src="images/exhibition.png" alt="" class="prog-img">
+          </div>
+          <div class="topic">
+            <h4 class="clr-2 head4">Workshop</h4>
+          </div>
+          <p class="desc">
+            To provide an opportunity for the financial assistance.
+          </p>
+        </div>
+        <div class="pro-item">
+          <div class="prog-img-ph">
+            <img src="images/roulette.png" alt="" class="prog-img">
+          </div>
+          <div class="topic">
+            <h4 class="clr-2 head4">Ignite</h4>
+          </div>
+          <p class="desc">
+            The skills and Management you need to enage successfully with Trading partners.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="btn clr-4" id="join">
+        SEE THE WHOLE PROGRAM
+      </button>
+    </section>
+
+  </main>
+  <footer class="footer" id="footer">
+    <div class="container-main" id="partners-ph">
+      <h3 class="head3 clr-4 fw" id="partners">Partners</h3>
+      <hr>
+      <div class="partners-list">
+        <div class="partners">
+          <img src="images/ec.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/offsec2.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/microsoft-r-logo-png-12.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/cisco.png" alt="">
+        </div>
+        <div class="partners">
+          <img src="images/mcAfee.png" alt="">
+        </div>
+      </div>
```

Comment:
- [ ]  As it is provided from the design you should not have the footer part on the mobile version. Please check the provided behance design imrove your design as well. 

Your design 

![image](https://user-images.githubusercontent.com/43877689/227646942-b2a8baaf-cd20-495a-bae3-544d8bc69d8f.png)

Behance design

![image](https://user-images.githubusercontent.com/43877689/227647082-924c1f26-2814-43e3-a573-d41afe0a4629.png)

Reply:
Thank you so much for finding out my mistakes .